### DISCUSSION TITLE
issues with simple trail/ visualization

### DISCUSSION THREAD

**Discussion by aovaaa** [visose/Robots#230] [Q&A] [answered]
Hi, I am working on an animation of a ur10 printing a t-shirt. The script works great however I run into trouble when visualizing the trail.  I have tried several ways without being able to solve this issue. 

Here is some example of what I tried so far: 

**1. Using simple trail:** but it's for some reason  not tracing the correct geometry 
**2.  a workaround using** deconstruct program/ split list: but because of what I think is  some errors in the polyline I can not construct a pipe from it. 

<img width="1287" alt="Screen Shot 2023-06-06 at 14 48 12" src="https://github.com/visose/Robots/assets/133864224/fe478e67-2b10-4b50-8318-01a68f1e4d0b">
<img width="1330" alt="Screen Shot 2023-06-06 at 13 42 30" src="https://github.com/visose/Robots/assets/133864224/43967e8d-c27e-4695-ae63-d2e025767605">

I am new to "robot" and have been struggling with this issue for a while so would be super thankful if anyone has an idea or tip for how to solve the issue

You can find the rhino / gh file here: 

https://drive.google.com/drive/folders/1vYrVbOu9ChfzJ_yNgoBev7duTakGxkHW?usp=sharing

thank you

**Comment by visose** [comment, marked as answer]
You might find some components for this here: https://github.com/visose/Extensions

**Reply by aovaaa** [reply]
amazing will deff. look into that thank you so much for the tip
